### PR TITLE
Tg2 55 board page board cards list

### PR DESCRIPTION
**Pull request by salitordjman**
[TG2-55](https://appleseedsstudents.atlassian.net/browse/TG2-55)
[TG2-74](https://appleseedsstudents.atlassian.net/browse/TG2-74)

### CODE REVIEW

**Comment by salitordjman** [comment]
[TG2-55](https://appleseedsstudents.atlassian.net/browse/TG2-55)

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+      <Router>
       <Navbar />
-      <PageDesign />
+      
+      <Routes>
+
+      <Route path="/" element={<PageDesign />} />
+      <Route path="board/:idProject" element={<PageBoard />} />
+
+
```
Comment (Toam10) [inline frontend/src/App.tsx]: remove empty line spaces

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import './PageBoard.css';
+
+function PageBoard() {
+  return (
+      <div>
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx]: remove unused div

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+          <div>
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx]: remove unused div

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
 import './App.css';
 import PageDesign from './Screens/GlobalComponents/Navbar/Components/PageDesign/PageDesign';
 import './Screens/GlobalComponents/Navbar/Navbar';
```
Comment (Toam10) [inline frontend/src/App.tsx:6]: if you dont use it remove it

File: frontend/src/App.tsx [hunk 5]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
 import './App.css';
```
Comment (Toam10) [inline frontend/src/App.tsx:4]: .css files should be in the bottom of the imports files

File: backend/src/controllers/tasks.controller.ts [hunk 6]
```diff
@@ -74,22 +85,54 @@ async function updateTask(req: Request, res: Response) {
...
+    const { idProject } = req.params;
 
-    res.status(200).send(`Deleted completed successfully, Deleted ${taskDetail.deletedCount} Tasks`);
+    const projectDetail = await Project.findById(idProject, { tasks: { $elemMatch: { _id: idTask } } });
+
+    if (!projectDetail) throw new Error('Not found a Project');
+    if (!projectDetail.tasks.length) throw new Error('Not found a Task');
+    if (!projectDetail.tasks[0].comments.length) throw new Error('Not found a Comment');
+
+    res.status(200).send(projectDetail.tasks[0].comments);
```
Comment (Toam10) [inline backend/src/controllers/tasks.controller.ts:135]: please do it like this 
const comments = projectDetail.tasks[0].comments
res.status(200).send(comments)

File: backend/src/controllers/tasks.controller.ts [hunk 7]
```diff
@@ -98,49 +141,76 @@ async function deleteAllTask(req: Request, res: Response) {
...
   } catch (error) {
     res.status(400).send((error as Error).message);
   }
 }
 
 async function deleteComment(req: Request, res: Response) {
   try {
+    const { idProject } = req.params;
     const { idTask } = req.params;
     const { idComment } = req.params;
```
Comment (Toam10) [inline backend/src/controllers/tasks.controller.ts:176]: please all the distracture do in one line

**Review by Toam10 — APPROVED** [review]

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx (hunk 2 above)
Comment (salitordjman) [reply frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx]: I was removed the "PageBoard.tsx" convet to "src//Board/Board.tsx"

**Comment by salitordjman** [comment]
[TG2-55](https://appleseedsstudents.atlassian.net/browse/TG2-55)

**Pull request merged**